community/samples/serving/helloworld-java-micronaut/src/main/java/com/example/helloworld/HelloWorldController.java

```diff
@@ -0,0 +1,18 @@
+package com.example.helloworld;
+
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+
+@Controller("/")
+public class HelloWorldController {
+
+    @Get(value = "/", produces = MediaType.TEXT_PLAIN)
+    public String index() {
+        String target = System.getenv("TARGET");
+        if (target == null) {
+            target = "NOT SPECIFIED";
+        }
+        return "Hello World: " + target;
```

Comment:
While I have no objection to this construction, the sample is currently following the previous hello world behavior.

The current model the "official" hello-world samples take is:

"Hello $TARGET!" with fallback to "Hello World!"

Pointing it out in case you prefer to align.